src/meta.ts

```diff
@@ -1 +1,8 @@
-export { name, version } from "../package.json";
+import { name as pkgName, version } from "../package.json";
+import { TSESLint } from "@typescript-eslint/utils";
+
+const meta: TSESLint.FlatConfig.PluginMeta = { name: pkgName, version };
```

Comment:
Why is this change necessary?
Without this change, there is no need to add `@typescript-eslint/utils` to the dependencies.

Reply:
Thanks a lot for the explanation.
Would it work if we change it to the following? If so, I prefer it that way.

```suggestion
const meta: { name: string, version: string } = { name: pkgName, version };
```